cpp/src/HitDigitizer.cc

```diff
@@ -66,15 +66,15 @@ void HitDigitizer::DigitizeTube(HitTube *aHT, PMTResponse *pr)
 
             isAlive = true;
             if (fApplyEff ){ this->ApplyThreshold(digiQ, isAlive); }
-            if( isAlive && nDigi==0 )
+            if( isAlive && nDigi==0 ) // why need nDigi==0? // want a digitizer deadtime?
```

Comment:
@akutsuR question for you (also L77 below)

Reply:
@pdeperio I do not remember the reason why I added "nDigi==0" to the condition at all... I beive that I wanted to do somthing with this when I did validation at the very early development stage.

Reply:
@kmtsui ok? can remove comment and resolve